private_gpt/server/chunks/chunks_service.py

```diff
@@ -24,17 +24,33 @@ class Chunk(BaseModel):
     document: IngestedDoc
     text: str = Field(examples=["Outbound sales increased 20%, driven by new leads."])
     previous_texts: list[str] | None = Field(
-        examples=[["SALES REPORT 2023", "Inbound didn't show major changes."]]
+        default=None,
+        examples=[["SALES REPORT 2023", "Inbound didn't show major changes."]],
     )
     next_texts: list[str] | None = Field(
+        default=None,
         examples=[
             [
                 "New leads came from Google Ads campaign.",
                 "The campaign was run by the Marketing Department",
             ]
-        ]
+        ],
     )
 
+    @classmethod
+    def from_node(cls: type["Chunk"], node: NodeWithScore) -> "Chunk":
```

Comment:
you couldn't use the Chunk type here? circular reference?

Reply:
The reason is that this class method is defined within Chunk class definition. So Chunk type is not ready.